### PR TITLE
Add "do not email" setting for subscriptions

### PR DESCRIPTION
**Pull request by nickpell**
http://manage.dimagi.com/default.asp?188535

@benrudolph @proteusvacuum @calellowitz cc: @kaapstorm

### CODE REVIEW

File: corehq/apps/accounting/forms.py [hunk 1]
```diff
@@ -362,6 +362,7 @@ class SubscriptionForm(forms.Form):
     no_invoice_reason = forms.CharField(
         label=ugettext_lazy("Justify why \"Do Not Invoice\""), max_length=256, required=False
     )
+    do_not_email = forms.BooleanField(label="Do Not Email", required=False)
```
Comment (proteusvacuum) [inline corehq/apps/accounting/forms.py]: Should this be translated?

**Comment by proteusvacuum** [comment]
:+1: 
Kicked build, fail looked unrelated.

**Comment by benrudolph** [comment]
:+1: pending farid's question

**Comment by nickpell** [comment]
I actually think not translating is the better thing to do, since these are Ops-only interfaces and adding translation tags only increases the burden on translators.

**Comment by benrudolph** [comment]
good point

**Comment by proteusvacuum** [comment]
:+1: Make sense.

**Pull request merged**